src/pages/Thanks/ThanksPage.vue

```diff
@@ -229,14 +226,9 @@ export default {
 				};
 				const limit = 1;
 
-				const valetInviterId = route?.query?.valet_inviter;
-
 				return Promise.all([
 					client.query({ query: experimentAssignmentQuery, variables: { id: 'share_ask_copy' } }),
 					teamId ? fetchGoals(client, limit, filters) : null,
-					!!valetInviterId && valetInviterId?.toUpperCase() !== KIVA_INVITER_ID
```

Comment:
Is there more code to remove related to this query?

Reply:
Looks like you already removed where `lenderPublicProfile` gets read from the apollo cache. Can `lenderPublicProfile` be removed now?

Reply:
Yup exactly, the query file.

Reply:
Oh good call, yeah go ahead and leave it.